src/ExNavigationBar.js

```diff
@@ -377,7 +378,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     borderBottomColor: ExNavigationBar.DEFAULT_BORDER_BOTTOM_COLOR,
     borderBottomWidth: ExNavigationBar.DEFAULT_BORDER_BOTTOM_WIDTH,
-    elevation: 2,
+    elevation: 4,
```

Comment:
While the default elevation of appbar is 4, it looks quite ugly with white background IMO.